ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh

```diff
@@ -37,6 +37,7 @@ cat > packet-setup.yaml <<-EOF
         plan: m2.xlarge.x86
         facility: any
         wait_for_public_IPv: 4
+        wait_timeout: 900
```

Comment:
The folks at Packet suggested 20-25mins.

Reply:
Increased to 20 mins (previous attempts didn't show an evident difference, but I guess it's still worth to have it set to a slightly larger value).